samples/tree-pollen/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// [START maps_tree_pollen]
+function getNormalizedCoord(coord, zoom) {
+    const y = coord.y;
+    let x = coord.x;
+    // tile range in one direction range is dependent on zoom level
+    // 0 = 1 tile, 1 = 2 tiles, 2 = 4 tiles, 3 = 8 tiles, etc
+    const tileRange = 1 << zoom;
+  
+    // don't repeat across y-axis (vertically)
+    if (y < 0 || y >= tileRange) {
+      return null;
+    }
+  
+    // repeat across x-axis
+    if (x < 0 || x >= tileRange) {
+      x = ((x % tileRange) + tileRange) % tileRange;
+    }
+    return { x: x, y: y };
+  }
+  
+  class AirQualityMapType {
+    tileSize;
+    alt = null;
+    maxZoom = 16;
+    minZoom = 3;
+    name = null;
+    projection = null;
+    radius = 6378137;
+    constructor(tileSize) {
+      this.tileSize = tileSize;
+    }
+  
+    getTile(coord, zoom, ownerDocument) {
+      const img = ownerDocument.createElement("img");
+      const mapType = "TREE_UPI";
+      const normalizedCoord = getNormalizedCoord(coord, zoom);
+  
+      const x = coord.x;
+      const y = coord.y;
+      const key = "AIzaSyD4jWorF4eIMiBE3oZnd73Y7kOVO9q6gBE";
```

Comment:
Just want to point out to whoever reviews this that there is an API key in here.